src/main/java/org/folio/orders/utils/validators/CompositePoLineValidationUtil.java

```diff
@@ -252,4 +255,15 @@ private static Error convertErrorCodesToError(CompositePoLine compPOL, ErrorCode
     }
     return error;
   }
+
+  private static List<ErrorCodes> validatePackagePoLine(CompositePoLine compPOL) {
+     List<ErrorCodes> errors = new ArrayList<>();
+
+     // The quantity of the electronic resources in the cost must be specified
```

Comment:
I think this comment is misleading. This method is not referencing any resources in cost?

Reply:
Side effect of copy-paste ) 
removed the comment